http/headers/Feature-Policy.json

```diff
@@ -141,7 +139,7 @@
               "webview_android": "mirror"
             },
             "status": {
-              "experimental": true,
+              "experimental": false,
```

Comment:
This is OK because the feature is available in multiple browsers. I suspect that this is why it was done as partial implementation.